tests/test_batch_run.py

```diff
@@ -0,0 +1,125 @@
+from mesa.agent import Agent
```

Comment:
Why aren't unittests being used like in other test files?

Reply:
With pytest there should be no need to import unittest. Was there anything in particular not working for you?

Reply:
Hmm it wasn't letting me run the tests, maybe I forgot a step setting up my ide since I just switched computers,  I will relook